RELEASE/scripts/autoscend/iotms/mr2022.ash

```diff
@@ -729,7 +741,8 @@ boolean auto_autumnatonQuest()
 
 boolean auto_hasSpeakEasy()
 {
-	return auto_is_valid($item[deed to Oliver\'s Place]) && get_property("ownsSpeakeasy").to_boolean();
+	// We aren't interested in using the deed, if it's unrestricted we should be able to reach the speakeasy
+	return is_unrestricted($item[deed to Oliver\'s Place]) && get_property("ownsSpeakeasy").to_boolean();
 }
```

Comment:
We don't need to use the deed to fight in the speakeasy in G-Lover, and couldn't use the deed if we wanted to.

I think this needs an additional check in KoE based on a comment in the discord, but I haven't tried running autoscend in that path yet.